tools/project_manager/manage_image_sources.php

```diff
@@ -490,10 +490,9 @@ function approve()
             $subject = sprintf(_('%1$s: Image Source %2$s has been approved!'),$site_abbreviation,$this->display_name);
 
             $body = "Hello $username,\n\n" .
-                "This is a message from the $site_name website.\n\n".
                 "The Image Source that you proposed, $this->display_name, has been\n".
                 "approved by $pguser. You can select it, and apply it to projects, from\n".
-                "your project manager's page.\n\n$site_signoff";
+                "your project manager's page.\n";
```

Comment:
add thank-you?

Reply:
I don't think this one really needs it. If we're going to add it to _every_ email we should just put it in `maybe_mail()`.